python/ray/tests/test_tls_auth.py

```diff
@@ -148,7 +148,9 @@ def test_client_connect_to_tls_server(use_tls, call_ray_start):
      """,
         env=tls_env,
     )
-    assert out.strip() == "True"
+    # ignore logger warning on sigterm handler
+    out_msg = out.strip().split("\n")[0]
+    assert out_msg == "True"
```

Comment:
I think we can do the check inside the driver script so we don't need to check out_msg. 

Concretely
```
 """
import ray
from ray.util.client import ray as ray_client
ray_client.connect("localhost:10001")
assert ray.is_initialized()
     """